ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__automated-release-stable-4.19-upgrade-from-stable-4.18.yaml

```diff
@@ -0,0 +1,102 @@
+base_images:
+  ansible:
+    name: "4.18"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.18"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.18"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.19"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.18"
+  tools:
+    name: "4.18"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.18"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+releases:
+  latest:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.18.0-0
+        upper: 4.19.0-0
+  target:
```

Comment:
@jhuttana 4.19 is not GA yet. it does not have build in release stream `4-stable`. we only can use ec build instead.
```
  target:
    release:
      architecture: amd64
      channel: candidate
      version: "4.19"
```

Reply:
xref: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-stable-4.19-upgrade-from-stable-4.18.yaml#L48-L52

Reply:
Done

Reply:
the update is incorrect. the expected content is mentioned above

Reply:
Yeah :) trying to include the exact same snippet in the next commit